configs/Asus/PRIME-X570-P.conf

```diff
@@ -0,0 +1,68 @@
+#
+# Manufacturer: ASUSTeK COMPUTER INC.
+# Product Name: PRIME X570-P
+# Super I/O: NCT6798D-R
+# BIOS Version: 1405
+#
+# Contributed by Alberto Pedron
+#
+
+chip "nct6798-*"
+
+  # Voltages
+  label in0 "VCore"
+  label in1 "+5V"
+  label in2 "AVSB"
+  label in3 "+3.3V"
+  label in4 "+12V"
+  label in7 "3VSB"
+  label in8 "VBAT"
+  label in9 "VTT"
+
+  label in5 "VIN8"
```

Comment:
Yes, these labels come form the Nuvoton NCT6796D datasheet. Unfortunately I was not able to find the NCT6798D one either but since the other voltages (VCORE, AVSB, 3VCC...) ended up to be in the same indexes as NCT6796D I assumed VINs were safe to label too. HWiNFO64 shows the same labels as well.
Anyway I realize this is mostly based on assumptions and I'm thinking about removing this section altogether. Let me know what do you think about it.